.github/workflows/update-search-index.yml

```diff
@@ -0,0 +1,43 @@
+name: Update Search Index
+    
+on:
+  push:
+    branches:
+      - main
+
+env:
+  NODE_VERSION: '20.x'
+
+jobs:
+  conditional-skip:
+    runs-on: ubuntu-latest 
+    name: Get files changed and conditionally skip updating the index
+    outputs:
+      trigger-ci: ${{ steps.read-files.outputs.trigger-ci }}
+    steps:
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v4.0.0
+        with:
+          fetch-depth: 0  
+      - name: Get changed files
+        id: read-files
+        run: ./.github/scripts/filter_changed_files.sh "website" "packages/flight-icons/catalog.json"
```

Comment:
we should look also at the design tokens (they're indexed too)

Reply:
do you have the path for that? should we also be triggering website CI for similar changes?

Reply:
Updated